crates/bevy_sprite/src/picking_backend.rs

```diff
@@ -30,6 +31,12 @@ pub enum SpritePickingMode {
 #[derive(Resource, Reflect)]
 #[reflect(Resource, Default)]
 pub struct SpritePickingSettings {
+    /// When set to `true` sprite picking will only consider cameras and entities explicitly marked
+    /// with [`SpritePickable`]. `true` by default.
+    ///
+    /// This setting is provided to give you fine-grained control over which cameras and entities
+    /// should be used by the sprite picking backend at runtime.
+    pub require_markers: bool,
```

Comment:
I don't like this field: it adds a surprising behavior change with little benefit. We should pick a standard option, and then stick to it.